updatecli/updatecli.d/bump-aliases.yml

```diff
@@ -88,7 +88,7 @@ sources:
       key: .build_id
     transformers:
       - findsubmatch:
-          pattern: "([0-9].[0-9].[0-9])-(.*)"
+          pattern: '(\d+.\d+.\d+)-(.+)'
           captureindex: 1
```

Comment:
That's also my understanding but https://github.com/updatecli/updatecli/blob/5bf98b41ed313f713c35d03dba68a8045c0d6096/pkg/core/transformer/main_test.go#L331 used `.` ... so I tested it out and it worked fine. That's the reason I didn't change it. In any case, this will be deprecated sooner than later